src/pages/ProductListingPage/ProductListing.jsx

```diff
@@ -1,8 +1,140 @@
 import React from 'react';
 import './ProductListing.css';
+import { Products } from './Products';
+
 
 export default function ProductListing() {
   return (
-    <div>ProductListing</div>
+    <>
+    <div className="cycles-main-container">
+       <div className="filter-container">
+           <h2 className="filter-name">FILTER</h2>
+           <div className="filters">
+
+           <div className="filter range">
+               <h2>Price Range</h2>
+               
+                <input type="range" name="cycle-range" id="cycle-range" min='1000' max='30000' />
+           </div>
+           
+
+           <div className="filter sort-filter">
+               <h3>Sort</h3>
+               <div className="radio-input">
+                <input type="radio" name="sort"/>
+                <label for="low-to-high">low-to-high</label>
```

Comment:
In jsx we use `htmlFor` instead of  `for` attribute 
and you missed to add `id` in `input` element